tests/integration/components/http-errors-code-test.ts

```diff
@@ -0,0 +1,107 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import Service from '@ember/service';
+// @ts-ignore
+import { setupIntl } from 'ember-intl/test-support';
+import sinon from 'sinon';
+
+class CSChat extends Service {
+  openTicket = sinon.stub();
+}
+
+module('Integration | Component | http-errors-code', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+
+  hooks.beforeEach(function () {
+    this.owner.register('service:cs-chat', CSChat);
+    this.csChat = this.owner.lookup('service:cs-chat');
+  });
+
+  ['404', '500'].forEach((statusCode) => {
+    test('it renders w/ the right title & subtitle', async function (assert) {
+      this.httpError = statusCode;
+
+      await render(hbs`<HttpErrorsCode @httpError={{this.httpError}} />`);
+
+      assert.dom('[data-control-name="http-error-code-title"]').hasText(this.intl.t(`errors.${statusCode}.title`));
+      assert
+        .dom('[data-control-name="http-error-code-subtitle"]')
+        .hasText(this.intl.t(`errors.${statusCode}.subtitle`));
+    });
+  });
+
+  test('When 404: the right hints are displayed', async function (assert) {
+    await render(hbs`<HttpErrorsCode @httpError="404" />`);
+    assert.dom('[data-control-name="http-error-code-hints"] > div').exists({ count: 2 });
+    assert.dom('[data-control-name="http-error-code-hints"] > div:first-child i.far').hasClass('fa-unlink');
+    assert
+      .dom('[data-control-name="http-error-code-hints"] > div:first-child')
+      .hasText(this.intl.t('errors.404.hints.url_accuracy_check'));
+    assert.dom('[data-control-name="http-error-code-hints"] > div:last-child i.far').hasClass('fa-key');
+    assert
+      .dom('[data-control-name="http-error-code-hints"] > div:last-child')
+      .hasText(this.intl.t('errors.404.hints.authorization_check'));
+  });
+
+  test('When 404: the right actions', async function (assert) {
```

Comment:
the right actions are displayed

Also, nit: group tests with separate modules for 404 & 500 ?